app/actions/AssetActions.js

```diff
@@ -298,17 +296,16 @@ class AssetActions {
                 description: description,
                 extensions: {
                     reward_percent: createObject.reward_percent * 100 || 0,
-                    whitelist_market_fee_sharing: []
+                    whitelist_market_fee_sharing: [],
+                    taker_fee_percent: createObject.taker_fee_percent * 100 || 0
```

Comment:
I'd copied the same logic of reward_percent; hence not sure if reward_percent logic is correct.

I know that when switching off the entire *market fee* options; the operation will unset all to null; I don't think with existing UI we can unset those extensions individually; perhaps @sschiessl-bcp can help here.

Reply:
There is no problem for `reward_percent` because the default value is `0` if `reward_percent` is `null`. I'm not sure if it can be copied to `taker_fee_percent` since the logic is different. Waiting for @sschiessl-bcp.

Reply:
This is an unwanted opinion of the UI, if not set, it should be null and the default logic of core should be used, whatever that may be. 

So in both cases it seems correct to set it to "undefined"